app/deniedrefs/DeniedRefsRepo.scala

```diff
@@ -46,10 +46,13 @@ final class DeniedRefsRepo @Inject() (
     replaceIndexes = true
   ) {
 
+  import org.mongodb.scala.model.Projections._
+
   //TODO:x bring back the projection attribute so we don't fetch entire collection each time to
   // see what is the latest it
   def findLatestDeniedRefsId(): Future[Option[DeniedRefsId]] = collection
     .find()
+    .projection(slice("_id", 1))
```

Comment:
Should I close my pr?

Reply:
Maybe better I'll remove so it can be tested independently? I've added it by accident ... I was testing something.

Reply:
No worries